client/src/pages/users/CreateUser.js

```diff
@@ -106,16 +112,27 @@ const CreateUser = ({ triggerSearch, onClose, roles }) => {
 	return (
 		<>
 			<form>
-				<FormControl>
+				<FormControl isRequired isInvalid={!formData.fullName}>
 					<FormLabel htmlFor="fullName">Full Name</FormLabel>
 					<Input
 						id="fullName"
 						name="fullName"
 						aria-describedby="full name"
 						value={formData.fullName}
 						required
-						onChange={(e) => onChangeHandler(e)}
+						onChange={onChangeHandler}
 					/>
+					{!formData.fullName === "" ? (
```

Comment:
This needs to be redone as pointed out earlier in a similar case. Please revisit the logic of the query